### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by orchard0**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+function App() {
+	return (
+		<BrowserRouter>
+			<Header></Header>
+
+			<Routes>
+				<Route path="/" element={<Articles></Articles>}></Route>
+			</Routes>
+		</BrowserRouter>
+	);
```
Comment (Harrison365) [inline src/App.jsx:16]: Nice routes setup. You can put your BrowserRouter tags around your App tag in main.jsx to clean things up if you'd like.

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+	const getArticles = async () => {
+		try {
+			const results = await api.get('/articles');
+			setArticles(results.data.articles);
+		} catch (err) {
+			console.log(err);
+		}
+	};
+
```
Comment (Harrison365) [inline src/components/Articles.jsx:9]: Nice async await. However, your api-calling functions should be in a separate file and imported in for your useEffect

File: src/components/Articles.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	}, []);
+
+	return (
+		<>
+			{articles.map((article) => {
+				return (
+					<ArticleCard
+						key={article.article_id}
+						article={article}></ArticleCard>
+				);
```
Comment (Harrison365) [inline src/components/Articles.jsx:32]: Nice map 👍

File: src/components/Articles.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+			const results = await api.get('/articles');
+			setArticles(results.data.articles);
+		} catch (err) {
+			console.log(err);
+		}
+	};
+
+	useEffect(() => {
+		getArticles();
+	}, []);
```
Comment (Harrison365) [inline src/components/Articles.jsx:20]: Please also add some isLoading functionality so that users know that articles are still being fetched.

**Review by itskatherine — COMMENTED** [review]
Looks great Abdul, good to merge 👍

**Pull request merged**